requirements.txt

```diff
@@ -1 +1 @@
-sphinx==2.2.0
+sphinx>7.0.0
```

Comment:
Thank you, @runora95. I was reviewing the version of Sphinx used in our documentation projects, which ranges from `4.x` to `5.x`. Would upgrading to a version greater than `7.0.0` align with the repositories using this action? 

I'm new to this repository, but from what I understand, it's designed to notify authors about warnings in pull requests in advance. I'm just curious if the version mismatch could be a concern.

like in PennyLane : https://github.com/PennyLaneAI/pennylane/blob/master/doc/requirements.txt#L16

Reply:
Sounds good, thanks for the context!